Documentation/getting-started.md

```diff
@@ -36,6 +36,8 @@ For conceptual documentation (architecture, how-tos, etc.) most of the [document
 
 * WPF Applications crash with `System.TypeLoadException` when the Visual C++ Redistributable for Visual Studio 2017 is not installed. The latest version of VC++ redistributable can be obtained from [the Visual C++ downloads page](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads). This dependency will be removed prior to .NET Core 3.0 final release.
     * This is tracked by [#37](https://github.com/dotnet/wpf/issues/37).
+* WPF Applications crash with 'Module not found' with a stack originating from wpfgfx_cor3.dll (for an example, see [#167](https://github.com/dotnet/wpf/issues/167)).  This occurs on Windows 7 SP1, Windows Server 2008 R2, and Windows Server 2008 R2 SP1 machines where an updated containing d3d_compiler.dll has not been installed.  This is due to a [dependency added in .NET Framework 4.7](https://support.microsoft.com/en-us/help/4020302/the-net-framework-4-7-installation-is-blocked-on-windows-7-windows-ser) that is not currently redistributed with WPF for .NET Core 3.  The previous link for .NET 4.7 installation issues has appropriate fixes for this issue linked.  We will handle this dependency prior to .NET Core 3.0 final release.
```

Comment:
I would move details about OS's and reasons into the issue itself (top post). Let's keep here just "symptom" + link to the issue for details to avoid duplication.

Reply:
Sure, I already have that in #189 and the link exists in #167, so that should be enough information to ensure people are informed.

Reply:
I also re-worded things to be a bit more straightforward.